### DISCUSSION TITLE
Group Template

### DISCUSSION THREAD

**Discussion by jens853** [KelvinTegelaar/CIPP#2933] [Q&A]
Hello,

I have recently started using CIPP and i facing an issue with deploying Group Templates:
Failed to create group. * for *.onmicrosoft.com A value is required for property 'mailNickname' of resource 'Group'.

I have tried manually adding the following.
"mailNickname": "*",
"mailEnabled": false,
"securityEnabled": true,

Original JSON from CIPP:
{
  "Displayname": "GroupName",
  "Description": "Description of Group",
  "groupType": "generic",
  "MembershipRules": null,
  "allowExternal": null,
  "username": null,
  "GUID": "81d02dab-025a-4db8-8237-24c70c0f8af2"
}

Am i missing something ?

Thanks.

**Comment by KelvinTegelaar** [comment]
Support is available for sponsors via our helpdesk. :)
